src/library_dylink.js

```diff
@@ -615,8 +615,6 @@ var LibraryDylink = {
       if (firstLoad) {
         // alignments are powers of 2
         var memAlign = Math.pow(2, metadata.memoryAlign);
```

Comment:
Perhaps leave an assertion here that the alignment is at least stack alignment?

Reply:
But there is no need for it to me.  The alignment of these global is set in the binary to whatever alignment it needs to be.   It could be 1 and that would be fine.

The reason I believe we were rounding up to STACK_ALIGN.. is that so the resulting `__heap_base` would still be aligned to `STACK_ALIGN`, but `getMemory` was already taking care of that.